.github/workflows/lint-test.yaml

```diff
@@ -120,8 +121,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/rust-setup
-      - run: cargo x lint
-      - run: cargo xclippy --workspace --all-targets
+      - run: cargo clippy --workspace --all-targets
```

Comment:
we don't have lint anymore?

Reply:
just added the `      - uses: pre-commit/action@v3.0.0` step. can be run locally as well via `pre-commit run` but shouldn't be necessary since it'll be run automatically on commit via a git hook.